package.json

```diff
@@ -5,7 +5,8 @@
   "scripts": {
     "ng": "ng",
     "start": "ng serve",
-    "build": "ng build --aot --prod",
+    "buildprod": "ng build --aot --prod",
```

Comment:
Small, but can these be `build` and `build:staging` with the default build going for production?